### PR TITLE
Provide Dashboard application

### PR DESCRIPTION
**Pull request by m-czernek**

### CODE REVIEW

File: dashboard/backend/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const express = require('express');
```
Comment (jramcast) [inline dashboard/backend/index.js:3]: We should be consistent with string quoting (double or single). In my case I prefer double quotation.

Thinks like this are automatically validated with linters. So maybe it's a good opportunity to integrate  **eslint** in our JavaScript code.

File: dashboard/backend/lib/constants.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// Frontend service
+const FRONTEND_SERVICE = process.env.FRONTEND_SERVICE || "frontend"
+const FRONTEND_SERVICE_PORT = process.env.FRONTEND_SERVICE_PORT || "3000"
+// Main endpoint
+const FRONTEND_FQDN = `http://${FRONTEND_SERVICE}.${TARGET_PROJECT}.svc.cluster.local:${FRONTEND_SERVICE_PORT}/frontend`
+
+
+const HISTORY_DATA = {
+    "source": "USD",
+    "target": "EUR"
```
Comment (jramcast) [inline dashboard/backend/lib/constants.js:32]: We can get rid of quotes here

```suggestion
    source: "USD",
    target: "EUR"
```

File: dashboard/frontend/src/ComponentCard.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import { Button, Card, CardHeader, CardBody, TextContent,Text } from '@patternfly/react-core'
+import { RebootingIcon ,ErrorCircleOIcon, OkIcon } from '@patternfly/react-icons';
+
+class ComponentCard extends Component {
+  static defaultProps = {
+    name: "default name",
+    url: ""
```
Comment (jramcast) [inline dashboard/frontend/src/ComponentCard.js]: I see that down in the code, we use `endpoint`, not `url`. Should we change it?
```suggestion
    endpoint: ""
```

**Comment by m-czernek** [comment]
@jramcast thank you very much, I reviewed all your changes, changed dockerfiles, re-tested and pushed to our quay. Keeping this PR open until we merge the actual lab.

**Comment by m-czernek** [comment]
Merged lab. Merging this app.

**Pull request merged**